web-console/knikubevirt/Create-vm/step-2-networking/create-vm-networking.md

```diff
@@ -3,17 +3,20 @@
 ## Step 2: Networking
 
 ![Default](img/3-00.png)
```

Comment:
Hello! I've left this info on other PRs, so apologies if this is repetitive. In the wireframes we are mixing both old and new PatternFly blues, and I was wondering if it would be worth it to update the wires accordingly. For example, here is the PatternFly blue for buttons: https://www.patternfly.org/v4/documentation/react/components/button/

And the new PatternFly blue would also apply to link buttons like Cancel

Reply:
Thanks, updated.